### PR TITLE
fix(MCDU): fixes no ALTN handling on windpage and init

### PR DESCRIPTION
**Pull request by Maximilian-Reuter**
Fixes #[issue_no]

## Summary of Changes

This PR fixes the handling of having no alternate defined in a simbrief flight plan. 
Previously the init page would display a NOT IN DATABASE error when importing flight plans without alternate.
The DES wind page would also display NAN values for the alternate wind. 

Now the init page shows no error and the DES wind page doesn't show any alternate information at all according to honeywell manual

## References

image

## Additional context

Discord username (if different from GitHub): \_chaoz_

## Testing instructions

1. Set up A/C C/D 
2. Power up MCDU
3. Load flight plan without alternate
4. no errors should occur 
5. go to the DES winds page
6. no alternate field should exist
7. no entry into 1R should be accepted
8. Requesting winds should correctly import winds and not show any alternate information

For regression testing doing the same process but with an alternate defined should work as expected

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js [hunk 1]
```diff
@@ -112,17 +112,23 @@ class CDUWindPage {
             requestButton = "REQUEST [color]amber";
         }
 
-        let alternateCell = "[ ]°/[ ][color]cyan";
-
-        if (mcdu.winds.alternate != null) {
-            alternateCell = `${CDUWindPage.FormatNumber(mcdu.winds.alternate.direction)}°/${CDUWindPage.FormatNumber(mcdu.winds.alternate.speed)}[color]cyan`;
+        let alternatHeader = "";
```
Comment (BlueberryKing) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js:115]:
```suggestion
        let alternateHeader = "";
```

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js [hunk 2]
```diff
@@ -112,17 +112,23 @@ class CDUWindPage {
...
+        let alternateCell = "";
+        let altFLightlevel = "";
+
+        if (mcdu.altDestination) {
+            alternatHeader = "ALTN WIND";
```
Comment (BlueberryKing) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js:120]: Should probably say "ALTERNATE". See reference below.

**Review by BlueberryKing — CHANGES_REQUESTED** [review]
LGTM

**Review by Maximilian-Reuter — COMMENTED** [review]
github is stupid

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #7835 
Tier of Testing : 1
Date : 23/7/2023

Testing Process:
- Generate simbrief with no ALTN
- Observe wind page
- Tada!

Testing Results:
Passed

**Review by BlueberryKing — APPROVED** [review]
LGTM

**Pull request merged**